### DISCUSSION TITLE
Structure of Taxonomy

### DISCUSSION THREAD

**Discussion by animafps** [aiming-taxonomy/glossary#5] [Ideas]
What should be the structure of this project?

Some of my ideas
- keep this structure with the one readme and then classification headings then underneath concepts and their descriptions and aliases
- Lists of all concepts in separate markdown files per there classification
- Single readme listing all concepts in alphabetical order then underneath their descriptions, the classifications and aliases

Do we keep it in plain markdown or introduce a static site generator with this but that is a bit more work

**Comment by riddbtw** [comment]
Regardless of implementation I think separate .mds will make it very cumbersome, with no real advantage since discussion around ideas will happen in git before attempting to merge to master.  If we were expecting a constant stream of multiple concurrent edits to definitions (which would be difficult to manage with 1 readme) then it would be worth the time to split out.

Could you give an example of what you mean by a classification?

**Reply by animafps** [reply]
For a classifications be like
- fundamental ideas
- major aim types(tracking, clicking, target switching) prob better name for this
- Strafe aim movements
Etc
Just to classify things so can be more organised
And thr classifications themselfs can be discussed and created

**Reply by riddbtw** [reply]
I see.  Maybe start with arbitrary hierarchy (so, just definitions, alphabetical order makes sense) and determine if they need to be grouped from there.

**Comment by riddbtw** [comment]
As for markdown, strongly recommend just keeping it markdown, making it easy for reuse across platforms.
